bus-mapping/src/evm/opcodes/callop.rs

```diff
@@ -170,7 +171,7 @@ impl<const N_ARGS: usize> Opcode for CallOpcode<N_ARGS> {
             .unwrap_or(false);
         // TODO: What about transfer for CALLCODE?
         // Transfer value only for CALL opcode, insufficient_balance = false.
-        if call.kind == CallKind::Call && !insufficient_balance {
+        if call.kind == CallKind::Call && !insufficient_balance && is_depth_ok {
```

Comment:
set "!insufficient_balance && is_depth_ok" as "is_precheck_ok" similar to scroll/develop